api/agentpb/agent.proto

```diff
@@ -167,6 +167,14 @@ message StartActionRequest {
   // PTSummaryParams describes parameters for PT summary.
   message PTSummaryParams {}
 
+  // PTMySQLSummaryParams describes parameters for PT MySQL summary.
+  message PTMySQLSummaryParams {
+    string address = 1;
+    string port = 2;
```

Comment:
uint32

Reply:
Are you sure? It is defined as uint16 in the Service struct.

Reply:
I'm sure that there is no uint16 in protobuf :)

Reply:
ok